db/models/assessment.py

```diff
@@ -0,0 +1,64 @@
+import uuid
+
+from db import db
+from sqlalchemy.exc import IntegrityError
+
+
+class Assessment(db.Model):
+    id = db.Column(
+        "id",
+        db.Text(),
+        default=lambda: str(uuid.uuid4()),
+        primary_key=True,
+    )
+    application_id = db.Column(db.Text(), index=True, unique=True)
+
+    def __repr__(self):
+        return "<Assessment {} for Application {}>".format(
+            self.id, self.application_id
+        )
+
+    @property
+    def as_json(self):
```

Comment:
⛏️ How about just "json" since it's a property

Reply:
converted to normal method as maybe that's the more standard convention for a serialisation operation

Reply:
💯